src/language/class-modifiers.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Class modifiers
+description: Modifier keywords for class declarations to control external library access.
+---
+
+Class modifiers control how a class or a mixin can be subtyped, both
+[from within its own library](#abstract), and from outside of the library where
+it's defined.
+
+Modifiers prepend a class or mixin declaration.
+For example, writing `abstract class` defines an abstract class.
+The full set of modifiers that can appear before a class declaration are:
+
+- `abstract`
+- `base`
+- `final`
+- `interface`
+- `sealed`
+- [`mixin`][class, mixin, or mixin class]
+
+Only the `base` modifier can appear before a mixin declaration. The modifiers do
+not apply to other declarations like `enum`, `typedef`, or `extension`.
+
+When deciding whether to use class modifiers, consider the intended uses of the
+class, and what behaviors the class needs to be able to rely on.
+
+## No modifier
+
+Use a `class` or `mixin` declaration without a modifier to allow unrestricted
+permission to create subtype relationships from any library.
+By default, you can:
+
+- [Construct][] new instances of a class.
+- [Extend][] a class to create a new subtype.
+- [Implement][] a class or mixin's interface.
+- [Mix in][mixin] a mixin or mixin class.
+
+## `abstract`
+
+Use the `abstract` modifier to define a class that can’t be constructed from any
+library (it's own, as well as outside libraries).
```

Comment:
i would say the reason you use abstract isn't to prevent construction so much as to allow you to not provide a full concrete implementation of the entire interface.

Reply:
Changed to:
> Use the `abstract` modifier to define a class that doesn't require a full, concrete
implementation of the entire interface.